guide/2011504.md

```diff
@@ -5,18 +5,18 @@ title: Linking
 To link to another zettel, use the following syntax:
 
 ```markdown
-This is a zettel file, which links to another zettel: 
+This is a zettel file, which links to another zettel:
 
-- [2008403](z:). 
```

Comment:
Until the _full_ range of links (z:,  zcf:, zquery:, zcfquery:) get their alias syntax in neuron, docs should continue to recommend the old style. You can mention as an aside that "<1234567>" is an alternative syntax (with a note that it would be expanded as discussed [here](https://github.com/srid/neuron/issues/59)) for z:.